certified-connectors/SkyPointCloud/apiProperties.json

```diff
@@ -0,0 +1,38 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "CLIENTID",
+          "scopes": [
+            "https://sicdpstage.onmicrosoft.com/api/user_impersonation openid profile offline_access"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "publisher": "SkyPoint Cloud",
+            "stackOwner": "SkyPoint Cloud"
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://sicdpstage.b2clogin.com/sicdpstage.onmicrosoft.com/b2c_1a_accountlink_signup_signin/oauth2/v2.0/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://sicdpstage.b2clogin.com/sicdpstage.onmicrosoft.com/b2c_1a_accountlink_signup_signin/oauth2/v2.0/token"
+            },
+            "refreshUrl": {
+              "value": "https://sicdpstage.b2clogin.com/sicdpstage.onmicrosoft.com/b2c_1a_accountlink_signup_signin/oauth2/v2.0/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#fefefe",
```

Comment:
@vaibhavrmore1 Pure white color is not allowed in the background of the icon. 

Refer - [Documentation](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#icon)